ports/raspberrypi/boards/weact_studio_rp2350b_core/mpconfigboard.mk

```diff
@@ -0,0 +1,12 @@
+USB_VID = 0x2E8A
+USB_PID = 0x000F
```

Comment:
Hi, we need an allocated PID for this, ideally from WeAct itself (if it has a VID), or from https://github.com/raspberrypi/usb-pid. Failing that, it would be from https://pid.codes/ . `USB_PID = 000F` is the PID for the Pico 2. See https://learn.adafruit.com/how-to-add-a-new-board-to-circuitpython/customizing-the-board-files#usb-vid-and-pid-3137314, including the pointer to the FAQ for this.

Reply:
The `0x000F` PID is for the UF2 bootloader, but is not for when the board is running CircuitPython. You can't find the proper PID unless the manufacturer or someone else has registered a PID already. See https://learn.adafruit.com/how-to-add-a-new-board-to-circuitpython/frequently-asked-questions#faq-3130481.

I don't think WeAct has its own VID (vendor ID). They could ask Raspberry Pi for a PID (product ID) by going to https://github.com/raspberrypi/usb-pid. Or you or they could ask for one from https://pid.codes. Adafruit used to issue PID's to third parties but we stopped doing that.